app/src/main/java/com/example/tasky/itemdetail/presentation/EventDetailScreen.kt

```diff
@@ -0,0 +1,495 @@
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(horizontal = 16.dp),
+                            color = Light
+                        )
+
+                        Spacer(modifier = Modifier.size(30.dp))
+                    }
+
+                    item {
+                        Row(
+                            modifier = Modifier.padding(start = 16.dp),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            Text(
+                                text = stringResource(R.string.visitors),
+                                color = Black,
+                                fontWeight = FontWeight.Bold,
+                                fontSize = 20.sp
+                            )
+
+                            Spacer(modifier = Modifier.width(20.dp))
+
+                            if (state.isEditMode) {
+                                Box(
+                                    modifier = Modifier
+                                        .size(35.dp)
+                                        .clip(RoundedCornerShape(4.dp))
+                                        .background(Light2)
+                                        .clickable {
+
+                                        },
+                                    contentAlignment = Alignment.Center
+                                ) {
+                                    Icon(
+                                        modifier = Modifier.size(30.dp),
+                                        imageVector = Icons.Filled.Add,
+                                        tint = Gray,
+                                        contentDescription = null,
+                                    )
+                                }
+                            }
+                        }
+
+                        Spacer(modifier = Modifier.size(36.dp))
+                    }
+
+                    item {
+                        Row(
+                            modifier = Modifier.padding(horizontal = 16.dp),
+                            horizontalArrangement = Arrangement.spacedBy(16.dp)
+                        ) {
+                            VisitorTypePill(
+                                modifier = Modifier.weight(1f),
+                                visitorType = VisitorType.All(isSelected = true)
+                            ) {}
+                            VisitorTypePill(
+                                modifier = Modifier.weight(1f),
+                                visitorType = VisitorType.Going(isSelected = false)
+                            ) {}
+                            VisitorTypePill(
+                                modifier = Modifier.weight(1f),
+                                visitorType = VisitorType.NotGoing(isSelected = false)
+                            ) {}
+                        }
+
```

Comment:
If you migrate to Enum (explained on another comment)

You could ran a For loop on the VisitorType:

```
Row(modifier = Modifier.padding(horizontal = 16.dp), horizontalArrangement = Arrangement.spacedBy(16.dp) {
    VisitorType.values().forEach {
        VisitorTypePill(modifier = Modifier.weight(1f), isSelected = state.selectedFilter == it)) {}
    }
}
```